templates/index.html

```diff
@@ -9,250 +9,266 @@
+            </div>
+            <p>Configure the machine’s network interfaces with bridges, <abbr title="Virtual Local Area Network">VLAN</abbr>s, bonds and more.</p>
+            <p>Create advanced filesystem layouts with <abbr title="Redundant array of independent disks">RAID</abbr>, <abbr title="Block Cache">bcache</abbr>, <abbr title="Logical Volume Management">LVM</abbr> and more.</p>
+          </div>
+				</li>
+				<li class="p-matrix__item">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}5626be9e-chassis-mangement.svg" alt="Chassis icon" />
+              <h4 class="p-heading-icon__title"><a class="p-link--strong p-heading--five" href="https://docs.ubuntu.com/maas/2.2/en/intel-rsd"
+						   onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Chassis management' });" class="p-link--external">Chassis management&nbsp;&rsaquo;</a></h4>
+            </div>
+            <p>Full chassis convergence, including Cisco UCS, HP Moonshot, Intel RSD and more.</p>
+            <p>Dynamic hardware resource management with Intel RSD.</p>
+          </div>
+				</li>
+				<li class="p-matrix__item">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}4e80e2f4-network-management.svg" alt="Network management image" />
+              <h4 class="p-heading-icon__title"><a class="p-link--strong p-heading--five" href="/tour#manage-network"
+						   onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });">Network management&nbsp;&rsaquo;</a></h4>
+            </div>
+            <p>Observes and catalogs every IP address on the network (IPAM).</p>
+            <p>Built-in highly available DHCP (active-passive) and DNS (active-active).</p>
+          </div>
+				</li>
+				<li class="p-matrix__item">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}14939f84-devops-integration.svg" alt="DevOps image" />
+              <h4 class="p-heading-icon__title"><span class="p-heading--five">DevOps integration</span></h4>
+            </div>
+            <p>Integration with devops automation tools like <a href="http://conjure-up.io/" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'DevOps integration' });" class="p-link--external">conjure-up</a>, <a href="https://jujucharms.com/" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'DevOps integration' });" class="p-link--external">Juju</a>, Chef, Puppet, SALT, Ansible and more.</p>
```

Comment:
All links requires an ```aria-label``` to explain where the link takes you too. Also need to explain its an external link.